wp-content/themes/missilethreat/template-parts/entry-header.php

```diff
@@ -7,28 +7,137 @@
  * @since 1.0.0
  */
 
-$entry_header_classes = '';
 
+$page_for_posts = get_option( 'page_for_posts' );
 ?>
 
-<header class="single__header<?php echo esc_attr( $entry_header_classes ); ?>">
+<?php 
+if( is_single() && 'defsys' === get_post_type() || is_single() && 'missile' === get_post_type() ) { 
+	
+	$missile_short_name = get_field('missile_name');
 
-	<div class="single__header-wrapper">
+	?>
+	<header <?php post_class('single__header'); ?>>
 
+	<div class="single__header-wrapper">
 		<?php
-			missilethreat_posted_on();
+		get_template_part( 'template-parts/breadcrumbs' );
+
+		if ( $missile_short_name ) { ?>
+			<h1 class="single__header-title"><?php echo $missile_short_name; ?></h1>
+			<?php the_title( '<div class="single__header-excerpt">', '</div>' );
+		} 
+		
+		else {
+			the_title( '<h1 class="single__header-title">', '</h1>' );
+		} ?>
+
+		<div class="page__header-divider page__header-divider--thicc"></div>
```

Comment:
This should either a horizontal rule `<hr />` or done with a border or pseudo element.